sdk/cosmos/azure-cosmos/azure/cosmos/cosmos_client.py

```diff
@@ -244,6 +245,47 @@ def create_database(  # pylint: disable=redefined-builtin
             response_hook(self.client_connection.last_response_headers)
         return DatabaseProxy(self.client_connection, id=result["id"], properties=result)
 
+    @distributed_trace
+    def create_database_if_not_exists(  # pylint: disable=redefined-builtin
+        self,
+        id,  # type: str
+        populate_query_metrics=None,  # type: Optional[bool]
+        offer_throughput=None,  # type: Optional[int]
+        **kwargs  # type: Any
+    ):
+        # type: (...) -> DatabaseProxy
+        """
+        Create the database if it does not exist already.
+
+        If the database already exists, it is returned.
```

Comment:
Could we please elaborate on this to state that the returned resource may not match the supplied create parameters (i.e. `offer_throughput`).